contracts/strategies/allocation/AllocationWithVotingBase.sol

```diff
@@ -176,14 +185,13 @@ contract AllocationWithVotingBase is IAllocationStrategy, Ownable, Initializable
         if (applicationId == 0) {
             // Create a new application
             applicationId = ++_index;
-            application = Application({
-                identityId: identityId,
-                applicationId: applicationId,
-                recipient: recipient,
-                status: Status.PENDING,
-                metadata: metadata,
-                creator: _sender
-            });
+            application = applications[applicationId];
+            application.identityId = identityId;
+            application.applicationId = applicationId;
+            application.recipient = recipient;
+            application.status = Status.PENDING;
+            application.metadata = metadata;
+            application.creator = _sender;
```

Comment:
^ could we revert to the older format. Just to keep it consistent.
readability also is nicer on the eye in the older format

Reply:
cant when using mapping inside.